libs/gi-ui/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "gi-ui",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/gi-ui/src",
```

Comment:
Please add this to ignore-paths in https://github.com/frzyc/genshin-optimizer/blob/master/.github/workflows/deploy-sro-pr.yml